### PR TITLE
Pull/25

### PR DESCRIPTION
**Pull request by krasi-georgiev**
some improvment I suggested.

see the commit messages for explanation

### CODE REVIEW

File: config/prombench.yaml [hunk 1]
```diff
@@ -1,11 +1,9 @@
-#prombench cluster config
 zone: {{ .ZONE }}
+projectid: {{ .PROJECT_ID }}
 cluster:
   name: {{ .CLUSTER_NAME }}
-  initialclusterversion: 1.10.4-gke.2
```
Comment (sipian) [inline config/prombench.yaml:5]: I prefer keeping this line. If this is not provided GKE will use the latest stable version in the zone.

File: cmd/prombench/main.go [hunk 2]
```diff
@@ -18,42 +18,39 @@ func main() {
 	g := gke.New()
 	k8sGKE := app.Command("gke", `Google container engine provider - https://cloud.google.com/kubernetes-engine/`).
 		Action(g.NewGKEClient).
-		Action(g.ConfigParse)
+		Action(g.DeploymentsParse)
```
Comment (sipian) [inline cmd/prombench/main.go:21]: k8s has a deployment object but gke can be used to deploy any resource.
Using DeploymentsParse might cause some confusion. I prefer using ResourceParse, ResourceFiles, etc

**Comment by sipian** [comment]
Since prombench.yaml is not a cluster creation request but a node pool creation request, it needs to be modified

**Comment by sipian** [comment]
I'll make the above mentioned changes

File: config/prombench.yaml (hunk 1 above)
Comment (krasi-georgiev) [reply config/prombench.yaml:5]: prombench.yaml only creates nodepools would this matter in that case?

**Pull request merged**